File: Recorder.Core/Models/UploadStatus.cs
namespace Recorder.Core.Models
{
    /// <summary>
    /// Constants representing the upload status of a recording in the database.
    /// </summary>
    public class UploadStatus
    {
        public static readonly string Unknown = "unknown";
        public static readonly string Pending = "pending";
        public static readonly string Uploaded = "uploaded";
        public static readonly string Deleted = "deleted";

        private UploadStatus() { }
    }
}

File: tauri-app/src-tauri/src/models/upload_status.rs
/// Constants representing the upload status of a recording in the database.
///
/// Ported from Recorder.Core/Models/UploadStatus.cs
pub struct UploadStatus;

impl UploadStatus {
    pub const UNKNOWN: &'static str = "unknown";
    pub const PENDING: &'static str = "pending";
    pub const UPLOADED: &'static str = "uploaded";
    pub const DELETED: &'static str = "deleted";
}
